app/src/main/java/io/github/droidkaigi/confsched/fragment/SessionsTabFragment.java

```diff
@@ -160,13 +161,19 @@ public void onBindViewHolder(BindingHolder<ItemSessionBinding> holder, int posit
                 @Override
                 public void liked(LikeButton likeButton) {
                     session.checked = true;
-                    onLikeChanged(session, position);
+                    final int pos = holder.getAdapterPosition();
+                    if (pos != RecyclerView.NO_POSITION) {
```

Comment:
Yes. This is for making Lint happy :-) The lint complained `Do not treat position as fixed; only use immediately and call holder.getAdapterPosition() to look it up later.` and marked that as an error. So we cannot complete the compile unless it is fixed or we disable the lint.

See http://stackoverflow.com/questions/37498233/recyclerview-adapter-lint-error-do-not-treat-position-as-fixed